tree-sitter/src/CodeGen/GenerateSyntax.hs

```diff
@@ -34,6 +32,7 @@ import Data.Aeson hiding (String)
 import System.Directory
 import System.FilePath.Posix
 import TreeSitter.Node
+import TreeSitter.Symbol (escapeOperatorPunctuation)
```

Comment:
I really appreciate the explicit import of this one symbol here 👍 Nice.

Reply:
tyvm!